lib/utmp.c

```diff
@@ -203,7 +203,6 @@ get_session_host(char **out, pid_t main_pid)
 #if defined(HAVE_STRUCT_UTMPX_UT_HOST)
 	if ((ut != NULL) && (ut->ut_host[0] != '\0')) {
 		*out = XSTRNDUP(ut->ut_host);
-		free (ut);
```

Comment:
This is being fixed already: <https://github.com/shadow-maint/shadow/pull/1290/commits/a3667fdfd293270ab14a61657d8af2598f4d3364>

Reply:
Probably it worth to merge the fix separately from the larger PR as it looks like independent?

Reply:
You are welcome.
I'll update it based on your comments.